azure-storage-cpp-lite/src/storage_account.cpp

```diff
@@ -5,7 +5,8 @@
 namespace microsoft_azure {
     namespace storage {
 
-        storage_account::storage_account(const std::string &account_name, std::shared_ptr<storage_credential> credential, bool use_https, const std::string &domain_suffix)
+        // TODO: Clean up table queue and file services
+        storage_account::storage_account(const std::string &account_name, std::shared_ptr<storage_credential> credential, bool use_https, const std::string &blob_endpoint)
```

Comment:
oh, what a strange way to populate and handle those variables

Reply:
Are you saying 'passing by reference' is odd ? We can address this later... I am only adding blob_endpoint here..

Reply:
no -) i am saying that this class is doing a very simple thing in an unexpectedly complex way -) I ok with your changes.